SPECS/xapi.spec

```diff
@@ -1375,6 +1404,268 @@ Coverage files from unit tests
 %{?_cov_results_package}
 
 %changelog
+* Tue Apr 15 2025 Gaëtan Lehmann <gaetan.lehmann@vates.tech> - 25.6.0-1.1
```

Comment:
The fix from Mar to Apr should be made in the merge commit, not the in last commit